etc/kayobe/kolla.yml

```diff
@@ -271,11 +282,11 @@ kolla_enable_magnum: True
 #kolla_enable_manila_backend_hnas:
 #kolla_enable_mistral:
 #kolla_enable_monasca:
-kolla_enable_monasca: True
+kolla_enable_monasca: false
```

Comment:
Do we need to also disable other monasca services, to ensure they're not upgraded? kafka, grafana, zookeeper, kibana, influxdb, elasticsearch.